backend/src/fun_forecast_backend/main.py

```diff
@@ -1,21 +1,29 @@
-from typing import Any, Dict
+from dataclasses import asdict
+from typing import Any, Dict, List
 
 from fastapi import FastAPI
+from fastapi.middleware.cors import CORSMiddleware
 
 from fun_forecast_backend.activity_schema.activity import get_activity_schema_by_key
 from fun_forecast_backend.activity_schema.dataclasses import ActivitySchema
+from fun_forecast_backend.core.standard_logger import get_logger
+from fun_forecast_backend.forecast.dataclasses import Forecast
+from fun_forecast_backend.forecast.forecast import calculate_forecast
+from fun_forecast_backend.shared.dataclasses import HourData
+from fun_forecast_backend.weather.weather import hourly_forecast_for_coords
 
-app = FastAPI(title="Fun Forecast Backend")
+logger = get_logger()
 
-# app.add_middleware(
-#     CORSMiddleware,
-#     allow_origins=["*"],
-#     allow_credentials=True,
-#     allow_methods=["*"],
-#     allow_headers=["*"],
-# )
+app = FastAPI(title="Fun Forecast Backend")
 
-# logger = get_logger()
+app.add_middleware(
+    CORSMiddleware,
+    allow_origins=["http://myfunforecast.com", "http://beta.myfunforecast.com"],
```

Comment:
```suggestion
    allow_origins=["https://myfunforecast.com", "https://beta.myfunforecast.com"],
```
Secure protocol only